build.gradle

```diff
@@ -153,6 +153,12 @@ application {
     mainClass = 'bio.terra.cli.command.Main'
     applicationName = 'terra'
     executableDir = 'bin'
+
+    // TODO: Suppress Jersey PATCH related warnings (PF-622)
+    applicationDefaultJvmArgs = [
+            '--add-opens', 'java.base/sun.net.www.protocol.https=ALL-UNNAMED',
```

Comment:
It looks like this might be suppressing a pretty wide net. Can they be tightened to just the warnings we're seeing?

Reply:
No, I've never seen this before.

It would be great to suppress in the swagger builds for workspace-manager (and potentially others).